service/src/main/java/org/dice_research/fc/sparql/query/QueryEngineCustomHTTP.java

```diff
@@ -137,13 +141,25 @@ private String createRequest() {
                 .setConnectionRequestTimeout(timeout)
                 .setSocketTimeout(timeout).build();
         try(CloseableHttpClient client = HttpClientBuilder.create().setDefaultRequestConfig(config).build()){
+            LOGGER.info("--------Start Reqest------------");
+            LOGGER.info(service + "?query=" + URLEncoder.encode(query.toString(), "UTF-8"));
             HttpGet get = new HttpGet(service + "?query=" + URLEncoder.encode(query.toString(), "UTF-8"));
             get.addHeader(HttpHeaders.ACCEPT, "application/sparql-results+xml");
             HttpResponse resp = client.execute(get);
             String responseContent = read(resp.getEntity().getContent());
             InputStream is = new ByteArrayInputStream(responseContent.getBytes(StandardCharsets.UTF_8));
             String result = IOUtils.toString(is, StandardCharsets.UTF_8);
+
```

Comment:
add this because I face this error 
`2021-08-12 17:15:49,380 [http-nio-8282-exec-4] INFO [org.dice_research.fc.sparql.query.QueryEngineCustomHTTP] - <--------Start Reqest------------>
2021-08-12 17:15:49,380 [http-nio-8282-exec-4] INFO [org.dice_research.fc.sparql.query.QueryEngineCustomHTTP] - <https://synthg-fact.dice-research.org/sparql?query=SELECT++*%0AWHERE%0A++%7B+%3Chttp%3A%2F%2Fdbpedia.org%2Fontology%2Foffice%3E%0A++++++++++++++%3Chttp%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23domain%3E++%3Fx%7D%0A>
2021-08-12 17:15:49,518 [http-nio-8282-exec-4] INFO [org.dice_research.fc.sparql.query.QueryEngineCustomHTTP] - <<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
<html>
  <head>
    <title>Error HTTP/1.1 404 File not found</title>
  </head>
  <body>
    <h3>Error HTTP/1.1 404 File not found</h3><pre>
The requested URL was not found    URI  = '/sparql'
  </pre></body></html>
>
2021-08-12 17:15:49,518 [http-nio-8282-exec-4] INFO [org.dice_research.fc.sparql.query.QueryEngineCustomHTTP] - <----------end Reqest------------->`